.github/actions/python-bindings-action/action.yml

```diff
@@ -0,0 +1,75 @@
+name: Python bindings generation
+
+inputs:
+  USE_LAST_TAG:
+    description: Determines whether the last tag should be used in the pythonPack Gradle task
+    required: true
+
+runs:
+  using: composite
+  steps:
+    - name: Setup Python
+      uses: actions/setup-python@v5
+      with:
+        python-version: '3.13.2'
+
+    - name: Setup SWIG
+      shell: bash
+      run: sudo apt-get update && sudo apt-get install -y swig
+
+    - name: Setup Mingw
+      shell: bash
+      run: sudo apt-get install -y swig mingw-w64
+
+    - name: Setup GCC
+      shell: bash
+      run: sudo apt-get install -y swig mingw-w64 gcc g++
+
+    - name: Setup python build
+      shell: bash
+      run: python -m pip install build
+
+    - name: Generate
+      shell: bash
+      run: ./gradlew mingwX64Binaries
+
+    - name: Adjust header
+      shell: bash
+      run: |
+        sed -i \
+          -e 's/_\$$/_/g' \
+          -e 's/\$//g' \
+          "doordeck-sdk/build/bin/mingwX64/releaseShared/Doordeck.Headless.Sdk_api.h"
+
+    - name: Generate pyproject file
+      shell: bash
+      run: |
+        if [ "${{ inputs.USE_LAST_TAG }}" = "true" ]; then
+          ./gradlew -Prelease.useLastTag=true pythonPack
+        else
+          ./gradlew pythonPack
+        fi
+
+    - name: Download python for windows
+      shell: bash
+      run: |
+        curl -LO https://www.python.org/ftp/python/3.13.2/python-3.13.2-embed-amd64.zip
+        mv python-3.13.2-embed-amd64.zip doordeck-sdk/build/bin/mingwX64/python
```

Comment:
I guess `3.13.2` might be worth putting into a variable so the setup-python and this ensure they are using the same version